unity-renderer/Assets/DCLServices/WearablesCatalogService/LambdasWearablesCatalogService.cs

```diff
@@ -397,5 +448,39 @@ private static bool FilterInvalidWearableItem(WearableItem item, int index, List
 
         private static (string paramName, string paramValue)[] GetWearablesUrlParams(IEnumerable<string> wearableIds) =>
             wearableIds.Select(id => ("wearableId", id)).ToArray();
+
+        private void AddCollectionIdsAndCollectionCategoryParams(List<(string name, string value)> queryParams,
+            List<string> collectionIds = null)
+        {
+            HashSet<string> collectionCategories = new HashSet<string>();
+
+            if (collectionIds != null)
+            {
+                bool IsThirdParty(string wearableId) =>
+                    wearableId.Contains("collections-thirdparty");
+
+                bool IsBaseWearable(string wearableId) =>
+                    wearableId.StartsWith("urn:decentraland:off-chain:base-avatars:");
+
+                if (collectionIds.Any(IsThirdParty))
+                    collectionCategories.Add("third-party");
+
+                if (collectionIds.Any(IsBaseWearable))
+                    collectionCategories.Add("base-wearable");
+
+                if (collectionIds.Any(s => !IsBaseWearable(s) && !IsThirdParty(s)))
+                    collectionCategories.Add("on-chain");
+
+                queryParams.Add(("collectionIds", string.Join(",", collectionIds)));
+            }
+            else
+            {
+                collectionCategories.Add("third-party");
+                collectionCategories.Add("base-wearable");
+                collectionCategories.Add("on-chain");
```

Comment:
all those string could be constants

Reply:
I did change the implementation, which contains no repetitions of these strings.. do you find it necessary?